src/init.cpp

```diff
@@ -465,7 +466,7 @@ std::string HelpMessage(HelpMessageMode mode)
     {
         strUsage += HelpMessageOpt("-limitfreerelay=<n>", strprintf("Continuously rate-limit free transactions to <n>*1000 bytes per minute (default: %u)", DEFAULT_LIMITFREERELAY));
         strUsage += HelpMessageOpt("-relaypriority", strprintf("Require high priority for relaying free or low-fee transactions (default: %u)", DEFAULT_RELAYPRIORITY));
-        strUsage += HelpMessageOpt("-maxsigcachesize=<n>", strprintf("Limit size of signature cache to <n> MiB (default: %u)", DEFAULT_MAX_SIG_CACHE_SIZE));
+        strUsage += HelpMessageOpt("-maxsigcachesize=<n>", strprintf("Limit total size of signature and bundle caches to <n> MiB (default: %u)", DEFAULT_MAX_SIG_CACHE_SIZE));
```

Comment:
```suggestion
        strUsage += HelpMessageOpt("-maxsigcachesize=<n>", strprintf("Limit total size of signature caches and of bundle caches to <n> MiB each (default: %u)", DEFAULT_MAX_SIG_CACHE_SIZE));
```

Reply:
In fact, the current 1/2 - 1/4 - 1/4 split is also somewhat wasteful, as we'll likely never have 262144 bundles in the mempool at once. But if we stick with this flag as the single "control the max memory used for validation caches" (which [upstream also did when they added script validity caching](https://github.com/bitcoin/bitcoin/commit/b5fea8d0ccc8117644a4ea11256bc531b60fd9a3)) then it's simple for users to reason about, and we can tweak the internal distribution as we need.

Reply:
Ah ok, I misread the initialization calls.